django/contrib/admin/templates/admin/auth/user/add_form.html

```diff
@@ -3,9 +3,9 @@
 
 {% block form_top %}
   {% if not is_popup %}
-    <p>{% translate 'First, enter a username and password. Then, you’ll be able to edit more user options.' %}</p>
+    <p>{% blocktranslate %}First, enter a valid {{ username }} and password. Then, you’ll be able to edit more user options.{% endblocktranslate %}</p>
```

Comment:
I'm not sure about "valid", since this is a user creation form. If a field is invalid I think we can let them know in the errors.

```suggestion 
    <p>{% blocktranslate %}First, enter the {{ username }} and password. Then, you’ll be able to edit more user options.{% endblocktranslate %}</p>
```
Maybe the definite particle might be safe here: as in "*the* username and password for the new user." I don't know how that would translate in other languages, though.

Reply:
I agree using "the" is nicer than my suggestion :+1:

Reply:
I think the current sentence is the best if we keep the static form.
However, I think the sentence feels awkward from the users point of view if the username field changes like the issue we are dealing with now.
So I think it's better to use dynamic values even if we consider translation.
How about the sentence that @claudep suggested?
```
First, fill the fields: {{ username }} and password.
```
If there is no problem with the translation, I think this sentence is also good.